exporter/cassandraexporter/exporter_logs_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package cassandraexporter
+
+import (
+	"github.com/gocql/gocql"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewCluster(t *testing.T) {
+	testCases := map[string]struct {
+		cfg                   *Config
+		expectedAuthenticator gocql.Authenticator
+	}{
+		"empty_auth": {
+			cfg:                   withDefaultConfig(),
+			expectedAuthenticator: nil,
+		},
+		"empty_username": {
+			cfg: withDefaultConfig(func(config *Config) {
+				config.Auth.Password = "pass"
+			}),
+			expectedAuthenticator: nil,
+		},
+		"empty_password": {
+			cfg: withDefaultConfig(func(config *Config) {
+				config.Auth.UserName = "user"
```

Comment:
If username is null or password is null, shouldn't collector validate this and throw an error to tell the users they should provide both in the same time?

Reply:
It is a good comment. It might be worth doing some additional checking here

Reply:
I did it